src/server-renderer.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import { StaticRouter } from 'react-router';
+import Invision from './invision/invision';
+import { store } from './invision/store';
+import { searchMovieAction } from './invision/store/actions/search-movie.action';
+
+function renderHTML(html, preloadedState) {
+  return `
+      <!doctype html>
+      <html>
+        <head>
+          <meta charset=utf-8>
+          <title>React Server Side Rendering</title>
+          <link href="/css/main.css" rel="stylesheet" type="text/css">
+        </head>
+        <body>
+          <div id="root">${html}</div>
+          <script src="/js/main.js"></script>
```

Comment:
You should put it after script with **window.PRELOADED_STATE** to get access to this variable in the react components.